apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.ts

```diff
@@ -53,6 +53,22 @@ router.get(
       assessmentQuestionId: res.locals.assessment_question.id,
     });
 
+    // Fetch instance questions for initial render
+    const instance_questions = await queryRows(
```

Comment:
Previously, this was fetched on the client only. Now, we can take advantage of our SSR Preact.